### DISCUSSION TITLE
[info] OpenJS - upcoming conference news

### DISCUSSION THREAD

**Discussion by jorydotcom** [fastify/fastify#2819] [General] [answered]
Wanted to share this FYI out to the fastify community to let ya’ll know that the OpenJS Foundation has opened its [call for speakers](https://openjsf.org/announcement/2021/01/11/openjs-world-2021-save-the-date/) for the upcoming virtual OpenJS World event on 9 June 2021. We would love to make sure the project is well represented in the conference content! If anyone is interested and has more questions, wants help writing an abstract, or wants to get involved with the CFP Reviews, email me at jory @ thestoryofjory dot com or ping me in Slack.

We’d also love your help getting the word out about the event - if you’d be up to putting a link or banner on the project website, tweeting from your project account, etc. we would be very grateful!

**Comment by mcollina** [comment, marked as answer]
Hi @jorydotcom! Sure thing, we'll send a few submissions!

As for publishing a link on the website... The best approach would be if you could send a PR to https://github.com/fastify/website.
